packages/share/example/lib/main.dart

```diff
@@ -83,4 +133,11 @@ class DemoAppState extends State<DemoApp> {
           )),
     );
   }
+
+  Future<File> _createTextFile(String filename, String content) async {
```

Comment:
This is failing the [analyze](https://github.com/flutter/plugins/pull/970/checks?check_run_id=372549971) test with:

```Checking project share...

  info - The class 'Future' wasn't exported from 'dart:core' until version 2.1, but this code is required to be able to run on earlier versions at example/lib/main.dart:137:3 - (sdk_version_async_exported_from_core)

1 issue found; analyzed 9 source files in 19.6s.



The following packages have analyzer errors (see above):
 * share```

Reply:
I hope with my new commit this should be resolved.